src/uwtools/tests/config/formats/test_ini.py

```diff
@@ -1,4 +1,4 @@
-# pylint: disable=duplicate-code
+# pylint: disable=duplicate-code,missing-function-docstring
```

Comment:
This is interesting since it's the only addition to the file...and the ones below. Is it necessary, or just here for consistency with others?

If it's the latter, do we want to consider adding a set of rules that only apply to the tests?